### DISCUSSION TITLE
What prevents quarto to work with any jupyter kernel ?

### DISCUSSION THREAD

**Discussion by statquant** [quarto-dev/quarto-cli#1571] [Q&A] [answered]
Hello, my understanding is that quarto only works with py, r, julia kernels when the engine is jupyter.
Can someone explain to me what prevents any supported kernel to work in quarto.
Typically I am interested in the q/kdb kernel.
Many thanks for the great tool.

**Comment by jjallaire** [comment, marked as answer]
It does actually work with any kernel (we just lean our documentation towards Python and Julia b/c they are so commonly used in scientific computing). I personally know of it working with the sas, bash, apl, and c kernels so I'd guess you won't have any trouble with q/kdb

**Reply by statquant** [reply]
Thanks for the prompt answer, I will revert/edit with a working (hopefully) example.
